Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/TranslatedLibrary.java

```diff
@@ -207,7 +209,15 @@ public ExpressionDef resolveExpressionRef(String identifier) {
     }
 
     public OperatorResolution resolveCall(CallContext callContext, ConversionMap conversionMap) {
-        return operators.resolveOperator(callContext, conversionMap);
+        OperatorResolution resolution = operators.resolveOperator(callContext, conversionMap);
+
+        // For backwards compatibility, a library can indicate that functions it exports are allowed to be invoked
+        // with fluent syntax. This is used in FHIRHelpers to allow fluent resolution, which is implicit in 1.4.
```

Comment:
> This is used in FHIRHelpers to allow fluent resolution, which is implicit in 1.4.
Should be 1.5 right?

Reply:
No, the implementation of FHIRPath support in 1.3 and 1.4 was done as implicit fluent invocation, so the comment is saying this feature (allowFluent) enables that implicit capability in 1.5 so it's backwards compatibe.

Reply:
Gotcha.